data_pipelines_cli/cli_commands/deploy.py

```diff
@@ -196,6 +201,12 @@ def _sync_bucket(self) -> None:
     required=False,
     help="Authorization OIDC ID token for a service account to communication with cloud services",
 )
+@click.option(
+    "--sync-bucket",
+    is_flag=True,
+    default=False,
```

Comment:
For backward compatibility it should be True by default, isn't it?

Reply:
The question at hand is whether or not to take UX into account. If a user desires to activate "sync bucket," no action is required. However, if the user wants to deactivate it, they must explicitly provide it as an argument. From my perspective, this process can be somewhat confusing.

Reply:
true, but still more confusing will be situation when after upgrading to new version everything stops working. I would assume that sync is by default on as deploy is mostly about this. So maybe instead let's call this flag --disable-bucket-sync ?

Reply:
I will push an update